### PR TITLE
Added advanced-guide.md for LXD

### PR DESCRIPTION
**Pull request by toby63**
I added an advanced-guide to content/lxd.

First commit:
- Added advanced guide in structure.json under "getting-started - OpenNebula
- added introduction and links
- and links inside getting-started-cli.md

Second commit:
- Added section for instance configuration
- and links inside getting-started-cli.md

### CODE REVIEW

**Comment by toby63** [comment]
More content will follow. See #413 for the status what is already done.

File: content/STRUCTURE.json [hunk 1]
```diff
@@ -140,6 +140,12 @@
      "menu": ["LXD", "Getting started - OpenNebula"],
      "generator": "markdown",
      "meta": {"input": "lxd/getting-started-opennebula.md"}},
+     
+    {"path": "/lxd/advanced-guide/",
+     "title": "LXD - Advanced Guide",
+     "menu": ["LXD", "Advanced Guide"],
```
Comment (stgraber) [inline content/STRUCTURE.json:146]: I think this should be capitalized as "Advanced guide" for consistency

**Comment by stgraber** [comment]
Looks good to me, just one tiny nit about the title :)

File: content/STRUCTURE.json (hunk 1 above)
Comment (toby63) [reply content/STRUCTURE.json:146]: Ok, I will change this.

**Comment by toby63** [comment]
Ready for merge.

**Pull request merged**